### PR TITLE
Replace Travis with GitHub Actions

### PR DESCRIPTION
**Pull request by ddeboer**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| Documentation   | 
| License         | MIT

#### What's in this PR?

Replace Travis setup with a GitHub Actions workflow.

#### Checklist

- [ ] Updated CHANGELOG.md to describe BC breaks / deprecations | new feature | bugfix
- [ ] Documentation pull request created (if not simply a bugfix)

#### To Do

- [ ] If the PR is not complete but you want to discuss the approach, list what remains to be done here

### CODE REVIEW

**Comment by dbu** [comment]
thanks a lot!

can you rename coverage to "coverage & static" or something like that? and run it on php 8.0? there is a gazillion errors about return types raised by phpstan, i think they would not occur on 8.0. 

then we can later separately address the 8.1 deprecations.

**Comment by ddeboer** [comment]
Done.

File: phpstan.neon [hunk 1]
```diff
@@ -0,0 +1,4 @@
+parameters:
+  ignoreErrors:
+    - message: '#Cannot call method debug\(\) on Psr\\Log\\LoggerInterface\|null#'
```
Comment (dbu) [inline phpstan.neon:4]:
```suggestion
    # The logger is initialized to NullLogger in the constructor of the FilesystemRecorder, but phpstan does not see that
    - message: '#Cannot call method debug\(\) on Psr\\Log\\LoggerInterface\|null#'
```

**Review by dbu — APPROVED** [review]

**Comment by dbu** [comment]
thanks a ton!

File: phpstan.neon (hunk 1 above)
Comment (ddeboer) [reply phpstan.neon:4]: 👍

**Comment by dbu** [comment]
i thought scrutinizer failed in your fork because it is not the same repository. still fails here. i wonder what we should do. not sure if we need scrutinizer, imho we could also just drop that step.

**Pull request merged**